doc/develop/api/overview.rst

```diff
@@ -376,3 +376,8 @@ current :ref:`stability level <api_lifecycle>`.
      - Stable
      - 1.0
      - 2.0
+
+   * - :ref:`input_api`
+     - Stable
```

Comment:
Newly introduced subsystem cannot be classified as stable. See [API Lifecycle](https://docs.zephyrproject.org/3.2.0/develop/api/api_lifecycle.html#stable) for more information.